doc/cli.markdown

```diff
@@ -982,45 +982,47 @@ increase verbosity
 
 # SSH
 
-## ssh [uuid]
+## ssh &#60;applicationOrDevice&#62; [serviceName]
 
-Warning: 'balena ssh' requires an openssh-compatible client to be correctly
-installed in your shell environment. For more information (including Windows
-support) please check the README here: https://github.com/balena-io/balena-cli
+This command can be used to start a shell on a local or remote device.
 
-Use this command to get a shell into the running application container of
-your device.
+If a service name is not provided, a shell will be opened on the host OS.
+
+If an application name is provided, all online devices in the application
+will be presented, and the chosen device will then have a shell opened on
+in it's service container or host OS.
+
+For local devices, the ip address and .local domain name are supported.
 
 Examples:
+	balena ssh MyApp
 
-	$ balena ssh MyApp
-	$ balena ssh 7cf02a6
-	$ balena ssh 7cf02a6 --port 8080
-	$ balena ssh 7cf02a6 -v
-	$ balena ssh 7cf02a6 -s
-	$ balena ssh 7cf02a6 --noninteractive
+	balena ssh f49cefd
+	balena ssh f49cefd my-service
+	balena ssh f49cefd --port <port>
+
+	balena ssh 192.168.0.1 --verbose
+	balena ssh f49cefd.local my-service
+
+Warning: 'balena ssh' requires an openssh-compatible client to be correctly
+installed in your shell environment. For more information (including Windows
+support) please check the information here:
+	https://github.com/balena-io/balena-cli/blob/master/INSTALL.md#additional-dependencies
```

Comment:
NIT:
From: `For more information (including Windows support) please check the information here:`
To: `For more information (including Windows support) please check:`

No need to change and re-push this PR, this can be changed in any other future PR (if it was to be changed at all).

Reply:
Changed.